### ISSUE TITLE
schematics generating a wrong npm command

### ISSUE THREAD

**Issue by ricardo-mello** [scullyio/scully#1288]
### Description
If you create a new scully website, the `npm run scully` script fails. If no one is working on it, please let me know so I can submit a fix.

#### 🔬 Minimal Reproduction
On terminal:
- `ng new blog`
- `ng add @scullyio/init`
- `ng build --prod`
- `npm run scully`

#### 💻Your Environment

**Angular Version:**

<pre><code>Angular CLI: 11.0.7
Node: 14.15.4
OS: darwin x64</code></pre>

**Scully Version:**

<pre><code>  "@scullyio/init": "^1.1.1",
  "@scullyio/ng-lib": "^1.0.0",
  "@scullyio/scully": "^1.0.0",
</code></pre>

**Comment by flack** [comment]
I have the same problem on a fresh angular project. This is the output I get:

```
$ npm run scully

> my-app@0.0.0 scully /home/flack/git/my-app
> npm scully --


Usage: npm <command>

where <command> is one of:
    access, adduser, audit, bin, bugs, c, cache, ci, cit,
    clean-install, clean-install-test, completion, config,
    create, ddp, dedupe, deprecate, dist-tag, docs, doctor,
    edit, explore, fund, get, help, help-search, hook, i, init,
    install, install-ci-test, install-test, it, link, list, ln,
    login, logout, ls, org, outdated, owner, pack, ping, prefix,
    profile, prune, publish, rb, rebuild, repo, restart, root,
    run, run-script, s, se, search, set, shrinkwrap, star,
    stars, start, stop, t, team, test, token, tst, un,
    uninstall, unpublish, unstar, up, update, v, version, view,
    whoami

npm <command> -h  quick help on <command>
npm -l            display full usage info
npm help <term>   search for help on <term>
npm help npm      involved overview

Specify configs in the ini-formatted file:
    /home/flack/.npmrc
or on the command line via: npm <command> --key value
Config info can be viewed via: npm help config

npm@6.14.8 /usr/share/npm

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! my-app@0.0.0 scully: `npm scully --`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the my-app@0.0.0 scully script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/flack/.npm/_logs/2021-03-11T15_37_41_987Z-debug.log
```

**Comment by dlewisharris** [comment]
Having the exact same issue and error out as @flack also

**Comment by quentinhayot** [comment]
I have the same issue when trying to add scully to an existing angular app

**Comment by quentinhayot** [comment]
I was able to make it work by changing the package.json as follow:
```
    "scully": "npm scully -- --project myproject",
    "scully:serve": "npx scully serve -- --project myproject"
```
to
```
    "scully": "npx scully --project myproject",
    "scully:serve": "npx scully serve --project myproject"
```

**Comment by dlewisharris** [comment]
@quentinhayot Thank you, this worked for me in the end. I also think my issue was having a _dash_ in project name too. I renamed all instances my project in angular.json, deleted the node_modules folder, ran `npm install` then `npx scully --project` and it worked

**Comment by quentinhayot** [comment]
My project name was "b2b". So no dash...

**Comment by parikatla95** [comment]
I am getting the same issue, had to manually change the npm script to npx scully --project <projectName>.

**Comment by unnhao** [comment]
> I am getting the same issue, had to manually change the npm script to npx scully --project .

You can just change to "scully".

**Comment by SanderElias** [comment]
This is fixed in #1294. When that PR is approved I'll merge it. It's already released to npm, so, this should be fixed by now
